proxy-node-shared/src/main/java/uk/gov/ida/notification/validations/Base64SamlValidator.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.ida.notification.validations;
+
+import org.apache.commons.lang3.StringUtils;
+import org.glassfish.jersey.internal.util.Base64;
+import org.opensaml.saml.saml2.core.impl.ResponseImpl;
+import uk.gov.ida.notification.saml.SamlParser;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class Base64SamlValidator implements ConstraintValidator<ValidBase64Saml, String> {
+
+    public static final int MinLength = 64;
+    public static final int MaxLength = 64000;
+
+    @Override
+    public void initialize(ValidBase64Saml constraint) { /* intentionally blank */ }
+
+    @Override
+    public boolean isValid(String value, ConstraintValidatorContext context) {
```

Comment:
Could probably be more descriptive than 'value' maybe 'encodedSaml'

Reply:
I see your point - although to me, it's just a String (or nothing) until we've validated it. Happy to change it if it's helpful to clarify.

Reply:
(done for all)